packages/delivery-service/migrations/migration_lock.toml

```diff
@@ -0,0 +1,3 @@
+# Please do not edit this file manually
+# It should be added in your version-control system (i.e. Git)
```

Comment:
do we have prisma at the ds ?  if not remove

Reply:
removed: fab74eb